EmulatorPkg/EmulatorPkg.dsc

```diff
@@ -539,7 +539,7 @@
   #
   # CLANGPDB tool chain depends on WIN_HOST_BUILD flag to generate the windows application.
   #
-  GCC:*_CLANGPDB_*_DLINK_FLAGS     = /ALIGN:4096 /FILEALIGN:4096
+  CLANGPDB: *_*_*_DLINK_FLAGS = /ALIGN:256 /FILEALIGN:4096

```

Comment:
Why did this change to 256? For EmulatorPkg, it doesn't matter so much, but general rule of thumb should be 4096. Also, you replaced one instance of GCC:*_CLANGPDB_*_DLINK_FLAGS for the WinHost build, but there are two more below, for setting /BASE. Do those need to be updated?